src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3832,6 +3832,115 @@ def test_sql_elastic_pool_maintenance(self, resource_group, resource_group_locat
                      JMESPathCheck('maintenanceConfigurationId', self._get_full_maintenance_id(self.MDB1))])
 
 
+class SqlServerTrustGroupsScenarioTest(ScenarioTest):
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location='westeurope', name_prefix='clitest')
+    def test_sql_server_trust_groups(self):
+
+        resource_prefix = 'sqlstg'
+
+        account = self.cmd('account show').get_output_in_json()
+
+        self.kwargs.update({
+            'loc': 'westeurope',
+            'vnet_name': 'stgCliTestVname',
+            'subnet_name': 'stgCliTestSubnet',
+            'nsg': 'stgCliTestNsg',
+            'route_table_name': 'stgCliTestRouteTable',
```

Comment:
could we use `self.create_random_name` to use random name for these resources?

Reply:
I could change it. Would it be okay to do that, once network preparer is ready, since it takes a lot of time to rerecord tests? If not, i will do it now :)

Reply:
I see.